samples/language/Namespaces.qs

```diff
@@ -8,13 +8,13 @@
 namespace MyQuantumApp {
 
     // The following `open` directive is used to import all types and callables declared in the
-    // Microsoft.Quantum.Arrays namespace.
-    open Microsoft.Quantum.Arrays;
+    // Microsoft.Quantum.Diagnostics namespace.
+    open Microsoft.Quantum.Diagnostics;
 	
     @EntryPoint()
     operation Main() : Result[] {
-        // `ConstantArray` is in the Microsoft.Quantum.Arrays namespace
-        let array = ConstantArray(3, true);
+        // `Message` is in the Microsoft.Quantum.Diagnostics namespace
+        Message("Hello World!");
```

Comment:
This is a weird one, because `Message` is technically in the `Microsoft.Quantum.Intrinsic` namespace and is another callable that is part of the prelude and always available. If you want something from `Microsoft.Quantum.Diagnostics` you should pick `DumpMachine` instead.